0-bootstrap/hub/2-services/argocd/instances/db2-configure.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+metadata:
+  name: db2-configure
+  annotations:
+    argocd.argoproj.io/sync-wave: "250"
```

Comment:
Careful, the app named "db2cluster-instance" has a sync wave of 290. You might want to set this to a number in the range (290.300]